python/lib/sift_py/data_import/csv.py

```diff
@@ -197,16 +187,20 @@ def is_uint64(n: pd.Series) -> bool:
             if i + 1 == time_column:
                 continue
 
-            inferred_dtype = pd.api.types.infer_dtype(df[df.columns[i]], skipna=False)
-            dtype = types.get(inferred_dtype)
-            if dtype is None:
+            raw_dtype = str(df[df.columns[i]].dtype)
+            if raw_dtype == "float64":
+                raw_dtype = "double"
```

Comment:
Why change this to `double`?

Reply:
https://github.com/sift-stack/sift/blob/python/ch10_upload_service/python/lib/sift_py/ingestion/channel.py#L187